### PR TITLE
Sprint: Improve GitHub Assignment

### PR DESCRIPTION
**Pull request by mattkrick**
it 
- [ ] does not let you add a repository unless you have admin rights to it
- [ ] does not let you join a personal repository unless you are a collaborator
- [ ] does not let you join an org repository (public/private) unless you are a member

### CODE REVIEW

**Comment by mattkrick** [comment]
@jordanh please reivew & merge after the hotfix

**Comment by jordanh** [comment]
# Test Cycle 1
Status: in progress

## Tests
it 
- [ ] does not let you add a repository unless you have admin rights to it
- [ ] does not let you join a personal repository unless you are a collaborator
- [ ] does not let you join an org repository (public/private) unless you are a member

File: src/server/graphql/mutations/createGitHubIssue.js [hunk 1]
```diff
@@ -40,18 +71,32 @@ export default {
...
     const assigneeProvider = providers.find((provider) => provider.userId === assigneeUserId);
     if (!assigneeProvider) {
-      const assigneeName = await r.table('TeamMember').get(assignee)('preferredName');
+      const assigneeName = await r.table('TeamMember').get(assigneeTeamMemberId)('preferredName');
       throw new Error(`Assignment failed! Ask ${assigneeName} to add GitHub in Team Settings`);
     }
+    const adminProvider = providers.find((provider) => provider.userId === adminUserId);
+    if (!adminProvider) {
+      // this should never happen
+      throw new Error('This repo does not have an admin! Please re-integrate the repo');
```
Comment (jordanh) [inline src/server/graphql/mutations/createGitHubIssue.js]: I'm glad you covered this case, just in case.

**Comment by jordanh** [comment]
All tests passed!

**Pull request merged**